sdk/tables/data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 12.1.1 (Unreleased)
+## 12.1.1 (2021-08-10)
+
+### Bugs Fixed
+
+- Update dependency of `@azure/core-xml`. [#16816](https://github.com/Azure/azure-sdk-for-js/pull/16816)
```

Comment:
it is a dev dependency, so I am not sure it shows up in the release notes. Should `core-xml` be moved to the `dependencies` section instead?

Reply:
Never mind, I was looking at core-client's package.json not tables 😄

Reply:
It is a `devDependency` in @azure/core-client but an actual dependency in Tables